src/ray/util/event.h

```diff
@@ -181,6 +191,7 @@ class RayEvent {
  private:
   rpc::Event_Severity severity_;
   std::string label_;
+  std::unordered_map<std::string, std::string> custom_fields_;
```

Comment:
will we add support for integral/double types? it will be useful for recording and analyze